app-server/app/src/main/scala/user/Routes.scala

```diff
@@ -0,0 +1,48 @@
+package com.azavea.rf.user
+
+import scala.concurrent.ExecutionContext
+
+import akka.http.scaladsl.server.Directives._
+import akka.http.scaladsl.server.PathMatchers.JavaUUID
+import akka.http.scaladsl.model.StatusCodes
+
+import com.azavea.rf.datamodel.latest.schema.tables.UsersRow
+
+import com.azavea.rf.utils.Database
+
+
+/**
+  * Routes for users
+  */
+trait UserRoutes {
+  def userRoutes()(implicit db:Database, ec:ExecutionContext) = (
+    pathPrefix("users") {
+      pathEndOrSingleSlash {
+        get {
+          onSuccess(UserService.getUsers) { resp => complete(resp) }
+        } ~
+        post {
+          entity(as[UsersRow]) {
+            user =>
```

Comment:
We don't really have a scala style guide, but I think I prefer new lines in most cases with brackets. So something like this:

```scala
user => onSuccess(UserService.createUser(db, ec, user)) {
    resp => complete((StatusCodes.Created, user))
}
```

At the very least, we should be consistent, especially with spaces surrounding brackets -- closing brackets in this PR aren't padded with whitespace but the opening of them is.